akka-cluster/src/main/scala/akka/cluster/ClusterHeartbeat.scala

```diff
@@ -93,7 +97,12 @@ private[cluster] final class ClusterHeartbeatSender extends Actor with ActorLogg
   val filterInternalClusterMembers: Member => Boolean =
     _.dataCenter == cluster.selfDataCenter
 
-  val selfHeartbeat = Heartbeat(selfAddress)
+  var sequenceNr = 0
+
+  def selfHeartbeat() = {
```

Comment:
change at the call site of this to use `()` and should be moved outside the foreach

Reply:
yep :+1: